packages/compass-indexes/src/components/create-index-actions/create-index-actions.tsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
-import { css, Banner, spacing, Button } from '@mongodb-js/compass-components';
+import {
+  css,
+  Banner,
+  spacing,
+  ModalFooterButton,
```

Comment:
why do we need this?

Reply:
The spacing of ConfirmationModal (and now FormModal) buttons is different than what we have in create index modal.

Reply:
I see, is it only limited to this dialog? Since this is a special case I'd keep it styled "manually" here, and we can tackle this in a follow-up PR rather than introducing this concept right away, feels like something that can be better thought through independently.